README.md

```diff
@@ -218,6 +221,58 @@ if (entity != null) {
 }
 ```
 
+Google Cloud DNS (Alpha)
+----------------------
+- [API Documentation][dns-api]
+- [Official Documentation][cloud-dns-docs]
+
+*Follow the [activation instructions][cloud-dns-activation] to use the Google Cloud DNS API with your project.*
+
+#### Preview
+
+Here are two code snippets showing simple usage examples from within Compute/App Engine. Note that you must [supply credentials](#authentication) and a project ID if running this snippet elsewhere.
+
+The first snippet shows how to create a zone resource. Complete source code can be found on
+[CreateAndListZones.java](./gcloud-java-examples/src/main/java/com/google/gcloud/examples/dns/snippets/CreateAndListZones.java).
+
+```java
+import com.google.gcloud.dns.Dns;
+import com.google.gcloud.dns.DnsOptions;
+import com.google.gcloud.dns.Zone;
+import com.google.gcloud.dns.ZoneInfo;
+
+Dns dns = DnsOptions.defaultInstance().service();
+String zoneName = "my_unique_zone";
```

Comment:
Totally fine if you think addressing the above comment is unnecessary, just thought I'd point it out in case it wasn't a conscious decision to not change it.
Also, from what I'm reading on the cloud docs, it seems like underscores aren't accepted in zone names.  Should we put in dashes instead of underscores?